erdjs/src/balance.ts

```diff
@@ -57,26 +57,29 @@ export class Balance {
     }
 
     /**
-     * Returns the raw representation of the balance object, that is, its big-integer form (as a string).
+     * Returns the string representation of the value (as eGLD currency).
      */
-    raw(): string {
-        return this.value.toString();
-    }
-
-    formatted(): string {
-        let padded = this.raw().padStart(DENOMINATION, "0");
+    toCurrencyString(): string {
+        let padded = this.toString().padStart(DENOMINATION, "0");
         let decimals = padded.slice(-DENOMINATION);
         let integer = padded.slice(0, padded.length - DENOMINATION);
         return `${integer}.${decimals} eGLD`;
```

Comment:
The `Balance` class might eventually be used for ESDT as well - if it's easy, add `eGLD` as a class constant and reference it here, easier to generalize later.

Reply:
Will do in a future PR, since now, that would also imply to change:

```
static eGLD(value: any): Balance
```

to:

```
static fromCurrency(value: any, ticker: CurrencyTicker): Balance
```

I've added JIRA task EN-8361. Is it all right?